serverless-resources.yml

```diff
@@ -173,6 +173,20 @@ rBuildsBatchJobDefinitionUbuntu2004:
       Image: "#{AWS::AccountId}.dkr.ecr.#{AWS::Region}.amazonaws.com/r-builds:ubuntu-2004"
     Timeout:
       AttemptDurationSeconds: 7200
+rBuildsBatchJobDefinitionUbuntu2204:
```

Comment:
Oh yeah, this reminds me, we also need to update serverless.yml and the builder/docker-compose.yml file.

https://github.com/rstudio/r-builds/blob/61a629916852cd066906c4caeaa1cba5748c601b/serverless.yml#L34-L60

And then I think we're good. I'm working on related CentOS 8 issues now, and can look at migrating to Rocky here.